fleece/cli/config/config.py

```diff
@@ -0,0 +1,318 @@
+        config = _decrypt_item(config, stage=args.stage, key='', render='ssm')
+        write_parameters(config['config'], config['keys'][args.stage],
+                         args.path, **boto_args)
+
+
+def parse_args(args):
+    parser = argparse.ArgumentParser(
+        prog='fleece config',
+        description=('Configuration management')
+    )
+    parser.add_argument('--config', '-c', default='config.yml',
+                        help='Config file (default is config.yml)')
+    parser.add_argument('--username', '-u', type=str,
+                        default=os.environ.get('RS_USERNAME'),
+                        help=('Rackspace username. Can also be set via '
+                              'RS_USERNAME environment variable'))
+    parser.add_argument('--apikey', '-k', type=str,
+                        default=os.environ.get('RS_API_KEY'),
+                        help=('Rackspace API key. Can also be set via '
+                              'RS_API_KEY environment variable'))
+    parser.add_argument('--environments', '-e', type=str,
+                        default='./environments.yml',
+                        help=('Path to YAML config file with defined accounts '
+                              'and stage names. Defaults to '
+                              './environments.yml'))
+    subparsers = parser.add_subparsers(help='Sub-command help')
+
+    import_parser = subparsers.add_parser(
+        'import', help='Import configuration from stdin')
+    import_parser.set_defaults(func=import_config)
+
+    export_parser = subparsers.add_parser(
+        'export', help='Export configuration to stdout')
+    export_parser.add_argument('--json', action='store_true',
+                               help='Use JSON format (default is YAML)')
+    export_parser.set_defaults(func=export_config)
+
+    edit_parser = subparsers.add_parser('edit', help='Edit configuration')
+    edit_parser.add_argument('--json', action='store_true',
+                             help='Use JSON format (default is YAML)')
+    edit_parser.add_argument(
+        '--editor', '-e', default=os.environ.get('FLEECE_EDITOR', 'vi'),
+        help='Text editor (defaults to $FLEECE_EDITOR, or else "vi")')
+    edit_parser.set_defaults(func=edit_config)
+
+    render_parser = subparsers.add_parser(
+        'render', help='Render configuration for a stage')
+    render_parser.add_argument('--json', action='store_true',
+                               help='Use JSON format (default is YAML)')
+    render_parser.add_argument('stage', help='Target stage name')
+    render_parser.set_defaults(func=render_config)
+
+    upload_parser = subparsers.add_parser(
+        'upload', help='Upload configuration to SSM')
+    upload_parser.add_argument('stage', help='Target stage name')
+    upload_parser.add_argument('--path', '-p', required=True,
+                               help='Parameter store root path')
+    upload_parser.add_argument('--erase', action='store_true',
```

Comment:
I could see this being useful when completely destroying a service as a way to properly clean things up and not leave sensitive values lingering. It might be useful to have a "Are you really sure you want to do this? /clippy" control here though to prevent accidental nuking.

Reply:
@larsbutler with the yoke style configs, nuking the lambda nukes the config as well, since it is inside it. The `--erase` option would address that need, though if we are going to version configs it becomes more complicated, maybe this command will need to take a version to nuke.